docs/dialect.md

```diff
@@ -173,3 +173,110 @@ Allowable date parts: year, quarter, month, dayofyear, day, weekday, week, hou
 
   **Join Capabilities**  <br/>
   Join usage is also defined by capabilities in the manifest file. See the Query section for join relation capabilities [here]({{ site.baseurl }}/docs/capabilities).
+
+### Boolean Support:
+  Some databases need to customize boolean support functions.  A common case is when a database lacks native boolean support. <br/>
+  `format-true` and `format-false` are used in predicate statements. <br/>
+   **format-true** <br/> 
+  ```xml 
+   <format-true value='(1=1)' />
+  ```
+  **format-false** <br/>
+  ```xml 
+  <format-false value='(1=0)' />
+  ```
+
+**Note: Changes in Tableau 2020.3**
+
+  With the release of Tableau 2020.3, `format-true` and `format-false` will have two parameters, literal and predicate. <br/>
+  Literals is used when a boolean is used as a value. Default to: `x ? 1 : 0` <br/>
+  Example usage: `SELECT 1 as literalBool` <br/>
+  Predicates is used when a boolean is used as a predicate. Default to: `isTrue ? (1=1) : (1=0)` <br/>
+  Example usage: `SELECT something as test WHERE (1=1)` <br/>
+  `value` attribute is used as `predicate` for backwards compatability <br/>
+  Example:
+  ```xml 
+    <format-true literal='TRUE' predicate='TRUE' />
+  ```
+  ```xml 
+    <format-false literal='FALSE' predicate='FALSE' />
+  ```
+
+   **format-bool-as-value** <br/>
+    Used in CASE statements. Determines whether the true or false case is used first. The fucntion is only used when `CAP_QUERY_BOOLEXPR_TO_INTEXPR` capiblity is set to yes. 
```

Comment:
"The function is only used when _the_ `CAP_QUERY_BOOLEXPR_TO_INTEXPR` _capability_ is set to yes."

Reply:
Fixed this.